components/announcement_bar/cloud_announcement_bar/user_limit_announcement_bar.tsx

```diff
@@ -174,4 +174,4 @@ class UserLimitAnnouncementBar extends React.PureComponent<Props> {
     }
 }
 
-export default withGetCloudSubscription(UserLimitAnnouncementBar);
+export default UserLimitAnnouncementBar;
```

Comment:
I think you should remove the import statement for `withGetCloudSubscription` too.